tests/components/interactiveDataSources/Scoller-test.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable react/jsx-indent */
+import React from 'react';
+import Scroller from 'components/interactiveDataSources/scroller/Scroller';
+import ScrollerOverlay from "components/interactiveDataSources/scroller/scrollerOverlay/ScrollerOverlay";
+import ScrollerOverlayCard from 'components/interactiveDataSources/scroller/scrollerOverlay/ScrollerOverlayCard';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { render, screen } from '@test-utils';
+import { mount } from 'enzyme';
```

Comment:
Is there a way to refactor this test and use React Testing Library?  We're not writing new tests with Enzyme because React 18 doesn't support it.  We're eventually migrating our Enzyme tests to Testing Library.